apps/mobile/src/components/Post/PostInput.tsx

```diff
@@ -0,0 +1,72 @@
+import clsx from 'clsx';
+import { useColorScheme } from 'nativewind';
+import { TextInput, View } from 'react-native';
+
+import colors from '~/shared/theme/colors';
+
+import { Typography } from '../Typography';
+
+type Props = {
+  value: string;
+  onChange: (newText: string) => void;
+};
+
+export function PostInput({ value, onChange }: Props) {
+  const { colorScheme } = useColorScheme();
+
+  const isTextTooLong = value.length >= 600;
+  const characterCount = value.length;
+
+  const handleChangeText = (newText: string) => {
+    if (newText.length <= 600) {
```

Comment:
I think there's a `maxlength` that we can set on the input / text area directly that will prevent the user from typing more 👀

Reply:
Damnn. Nice found @kaitoo1! Didn't know it exist